var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -78,10 +81,13 @@ def install(self, spec, prefix):
             ])
 
         # BLAS arguments require path to libraries
-        # FIXME : (blas / lapack always provide libblas and liblapack as aliases)
-        make_args.extend([
-            'BLAS=-lblas',
-            'LAPACK=-llapack'
-        ])
+        # FIXME: (blas/lapack always provide libblas and liblapack as aliases)
+        if '@4.5.1' in spec:
```

Comment:
essentially `BLAS=-lblas` and `LAPACK=-llapack` are link flags, as far as i can tell. So there should be no harm using something like `BLAS=-L/path/to/spack/opt/openblas/lib -lopenblas` and alike (exactly what `to_link_flags(spec['blas'].blas_shared_lib)` will resolve to). Moreover, this is more correct as otherwise suite-sparse may link against some other blas/lapack (e.g. system-provided).
But this can be left for another PR.

W.r.t. 4.5.3., can you please check if resulting libraries are linked against Spack's blas/lapack without those config options?